src/elo/utils/getSeasonNumber.js

```diff
@@ -0,0 +1,12 @@
+import seasonNumber from '../../models/seasonNumber.js';
+
+async function getSeasonNumber() {
+    try {
+        const returnedSeasonNumber = await seasonNumber.findOne({}).exec();
```

Comment:
Can you add a check here in case it doesn't exist? If it doesn't exist, let's just create one. This will be easier for people spinning up new instances of the server.